src/exec/Shelley.hs

```diff
@@ -81,10 +81,14 @@ mkParams rr cdb nconf nstate blockchainTime = NodeArgs
   , btime = blockchainTime
   , chainDB = cdb
   , blockProduction = Nothing
+  -- Don't automatically add a dummy genesis EBB, we'll add the real one
+  -- later on
+  , initChainDB = \_cfg _cdb -> return ()
   , blockFetchSize = nodeBlockFetchSize
   , blockMatchesHeader = nodeBlockMatchesHeader
   , maxUnackTxs = maxBound
   , maxBlockSize = MaxBlockBodySize maxBound
   , chainSyncPipelining = pipelineDecisionLowHighMark 200 300 -- TODO: make configurable!
-  , mempoolCap = MempoolCapacityBytes 128_000
+  , mempoolCap = MempoolCapacityBytesOverride
+      (MempoolCapacityBytes 128_000) -- TBD ???? should we override?
```

Comment:
Why override? I think the default `NoMempoolCapacityBytesOverride` is fine.

Reply:
Wasn't sure. It was always 128Mb but I don't know why that was chosen.